adam/curriculum/generate_segmentation_results.py

```diff
@@ -0,0 +1,196 @@
+import argparse
+import base64
+import json
+import logging
+import shutil
+from pathlib import Path
+import random
+from typing import Any, Mapping
+
+import cv2
+import numpy as np
+
+import requests
+import yaml
+from tqdm import tqdm
+
+API_ROUTE = "/instanceseg"
+DESCRIPTION_FILENAME = "description.yaml"
+INFO_FILENAME = "info.yaml"
+COLOR_SEED = 42
+LABEL_COLORS = np.random.uniform(0, 255, size=(50, 3))
+STEGO_MODEL = "stego"
+MASK_RCNN_MODEL = "rcnn"
+
+
+def stego_postprocessing(data: Mapping[str, Any], img: Any) -> Any:
+    # Comment so current unused data in the `data` map can be recalled in the future
+    # boxes, label = data["boxes"], data["labels"]
+    masks = np.array(data["masks"])
+
+    beta = 0.6
+    alpha = 1.0
+    gamma = 0.0
+    for mask in masks:
+        r_map = np.zeros_like(mask).astype(np.uint8)
+        g_map = np.zeros_like(mask).astype(np.uint8)
+        b_map = np.zeros_like(mask).astype(np.uint8)
+
+        # mask color assignment
+        color = LABEL_COLORS[random.randrange(0, len(LABEL_COLORS))]
```

Comment:
Is there a reason to use `randrange()` over deterministically indexing the label colors by (for example) the mask index? If so this should use a seed for picking the color.